Sketch.java

```diff
@@ -8,28 +8,61 @@
    */
   public void settings() {
 	// put your size call here
-    size(400, 400);
+    size(200, 250);
   }
 
   /** 
    * Called once at the beginning of execution.  Add initial set up
    * values here i.e background, stroke, fill etc.
    */
   public void setup() {
-    background(210, 255, 173);
+    background(50, 129, 168);
   }
 
   /**
    * Called repeatedly, anything drawn to the screen goes here
    */
   public void draw() {
-	  
-	// sample code, delete this stuff
-    stroke(128);
-    line(150, 25, 270, 350);  
 
-    stroke(255);
-    line(50, 125, 70, 50);  
+  //foreground settings
+    stroke(0,0,0);
+    strokeWeight(1);
+    fill(50, 168, 74);
+    rect((float) (width/-20), (float) (height/1.06), (float) (width/0.9), (height));
```

Comment:
You want to draw and demonstrate the usage of variables more than what you've done here.  While you are able to scale the grass, a better demonstration of variable use would be to scale the whole figure.